data/team.yaml

```diff
@@ -10,3 +10,7 @@
   github: tjouan
   name: Thibault Jouan
   twitter: tjouan
+- email: paul.debodinat@gmail.com
```

Comment:
You will need a Gravatar configured for this email in order to actually appear on our website :bust_in_silhouette:

Reply:
done !